component/main.jsonnet

```diff
@@ -25,8 +25,29 @@ local memberlist_secret = kube.Secret(params.speaker.secretname) {
   },
 };
 
+local configmap = kube.ConfigMap(params.configmap_name) {
+  metadata+: {
+    namespace: params.namespace,
+  },
+  data: {
+    config:
+      if std.objectHas(params, 'addresses') then std.manifestYamlDoc({
```

Comment:
```suggestion
      if std.objectHas(params, 'addresses') and params.addresses != null then std.manifestYamlDoc({
```

Reply:
`and` is no valid jsonnet syntax `&&` works